base/strings/string.jl

```diff
@@ -438,3 +449,25 @@ function repeat(s::String, r::Integer)
     end
     return out
 end
+
+"""
+    repeat(c::Char, r::Integer) -> String
```

Comment:
I couldn't find an open issue on this, so I filed one here https://github.com/JuliaLang/julia/issues/22804